end2end_test.go

```diff
@@ -1689,6 +1699,50 @@ func theProjectionIsNotCalled(arg1 string) error {
 	return fmt.Errorf("projection '%s' not found", arg1)
 }
 
+func anErrorIsReturned() error {
+	if !strings.Contains(errs.Error(), "provided x-update operation id") {
+		return fmt.Errorf("expected the error to contain: %s, got %s", "provided x-update operation id", errs.Error())
+	}
+	return nil
+}
+
+func theFieldShouldHaveTodaysDate(field string) error {
+
+	timeNow := time.Now()
+	todaysDate := timeNow.Format("2006-01-02")
+
+	switch dbconfig.Driver {
+	case "postgres", "mysql":
+		switch field {
```

Comment:
rather than do a switch on the field should you just access the field on the contentEntity directly? (perhaps make the returned content entity from created as a variable. This will make it easier to change the name of the field in the test without having to update the steps (when you see field name in quotes it's because the intent is to be able to change the test easily without having to update the steps)